root/report/AnnotationsArtists.js

```diff
@@ -40,7 +40,7 @@ const AnnotationsArtists = ({
       </li>
       <li>
         {texp.l('Generated on {date}',
-                {date: formatUserDate($c.user, generated)})}
+                {date: formatUserDate($c, generated)})}
       </li>
```

Comment:
This kinda seems like it could be a shared component.

Reply:
Agreed, not sure I want to change it all in this PR though :)

Reply:
fI'll send a second PR on top of this for it :)